Core/GDCore/IDE/Project/ProjectResourcesCopier.cpp

```diff
@@ -25,28 +25,29 @@ bool ProjectResourcesCopier::CopyAllResourcesTo(
     bool updateOriginalProject,
     bool preserveAbsoluteFilenames,
     bool preserveDirectoryStructure) {
+
+  gd::Project& project = originalProject;
+  if (!updateOriginalProject) {
+    std::shared_ptr<gd::Project> clonedProject(new gd::Project(originalProject));
+    project = *clonedProject;
+  }
+

```

Comment:
This seems incorrect and is probably linked to issue you may have encountered:

A reference can't be changed after it's set. So when you do `gd::Project& project = originalProject`, any further assignement will change the originalProject. I.e: when you do `project = *cloneProject`, you *modify* originalProject to be equal to `clonedProject` (which is itself made from originalProject).

When a reference is first assigned, it says to which object it points to. But then when you change it, it modifies the object itself, not the reference.